sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs

```diff
@@ -38,6 +38,13 @@ public enum ServiceVersion
         /// </summary>
         internal const ServiceVersion LatestVersion = ServiceVersion.V2019_05_06_Preview;
 
+        /// <summary>
+        /// The service version to use when creating continuation tokens that
+        /// can be passed between different client libraries.  Changing this
+        /// value requires updating <see cref="Azure.Search.Documents.Models.SearchContinuationToken"/>.
+        /// </summary>
+        internal const ServiceVersion ContinuationTokenVersion = ServiceVersion.V2019_05_06_Preview;
```

Comment:
Might we want to version this independently - maybe even monotonically (simple but effective)? Not a big deal, but seems more likely to cause some confusion if its version varies from a future service version. It's really just our own internal versioning anyway, so a monotonic version wouldn't have any unintended correlation to the api-version.

Reply:
Yep - we need a little more clarity around that.  Currently some of the languages are specifying the same version as their client but I don't think that scales well into the future.   I was thinking of using a constant service API version as a discriminator because we already have them, but `1, 2, 3, ...` would work as well.  We'll discuss it at one of the stand ups soon.